gostatsd.go

```diff
@@ -17,54 +17,65 @@ import (
 )
 
 var (
-	// BuildDate is the date when the binary was built
+	// BuildDate is the date when the binary was built.
 	BuildDate string
-	// GitCommit is the commit hash when the binary was built
+	// GitCommit is the commit hash when the binary was built.
 	GitCommit string
-	// Version is the version of the binary
+	// Version is the version of the binary.
 	Version string
 )
 
 const (
-	ParamVerbose    = "verbose"
+	// ParamVerbose enables verbose logging.
+	ParamVerbose = "verbose"
+	// ParamCPUProfile enables use of profiler and write results to this file.
 	ParamCPUProfile = "cpu-profile"
-	ParamJson       = "json"
+	// ParamJSON makes logger log in JSON format.
+	ParamJSON = "json"
+	// ParamConfigPath provides file with configuration.
 	ParamConfigPath = "config-path"
-	ParamVersion    = "version"
+	// ParamVersion makes program output its version.
+	ParamVersion = "version"
 )
 
+// EnvPrefix is the prefix of the inspected environment variables.
 const EnvPrefix = "GSD" //Go Stats D
 
 func main() {
-	err, v, version := setupConfiguration()
+	v, version, err := setupConfiguration()
```

Comment:
Yeah I should have caught it up during review. Placement of error is the reverse of Node.JS ;)
Funny how Node and Go were both created in 2009 and both eschew exceptions in favour of errors but put it at a different place :D